pkg/server/postgres/postgres.go

```diff
@@ -176,3 +187,92 @@ func (p *PostgresPlugin) DeleteDatabase(db *atlas.Database) error {
 	// delete database
 	return nil
 }
+
+func checkUsers(db *atlas.Database, sqldb *sql.DB) error {
+	//// check if users exist; verify their passwords and roles
+	dbUserMap := make(map[string]string)
+	var u string
+	//getting all users from the database
+	rows, err := sqldb.Query(`SELECT u.usename FROM pg_catalog.pg_user u`)
+	if err != nil {
+		return err
+	}
+	for rows.Next() {
+		if err := rows.Scan(&u); err != nil {
+			return err
+		}
+		dbUserMap[u] = "delete me"
+	}
+	glog.V(4).Info("Usersexist in database are", dbUserMap)
+	users := db.Spec.Users
+	glog.V(4).Info("Users from yaml are", users)
+
+	for _, user := range users {
+		if _, ok := dbUserMap[user.Name]; !ok {
+			err = createuser(user.Name, user.Role, user.Password, sqldb)
+			if err != nil {
+				return err
+			}
+		} else {
+			dbUserMap[user.Name] = "dont delete"
```

Comment:
dont delete ==> dont-delete. add '-' to make it as a single word

Reply:
done